Extension/test/unitTests/updowngrade.test.ts

```diff
@@ -60,8 +64,8 @@ suite("UpgradeDowngrade", () => {
                     name: insider2, assets: three_assets}, {
                     name: insider1, assets: three_assets}, {
                     name: release0, assets: three_assets}];
-
-                const userVersion: PackageVersion = new PackageVersion(release1);
+                // In internal testing, the name of the release has a "-master" at the end of it.
+                const userVersion: PackageVersion = new PackageVersion(release1 + "-master");
```

Comment:
For internal developers, the name of the branch has a "-master" at the end of it. I added this to the tests, and solved it by this check:
`(!testingInsidersVsixInstall && userVersion.suffix && userVersion.suffix !== 'insiders')`